File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/common/BaseResultData.java
package jp.co.terraresta.androidlesson.data.model.common;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by ooyama on 2017/05/26.
 */

public class BaseResultData {

    @Expose
    private int status;
    @SerializedName("error")
    @Expose
    private ErrorData errorData;

    public int getStatus() {
        return status;
    }

    public ErrorData getErrorData() {
        return errorData;
    }
}

File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/common/BaseResultData.kt
package jp.co.terraresta.androidlesson.data.model.common

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by ooyama on 2017/05/26.
 */

open class BaseResultData {

    @Expose
    val status: Int = 0
    @SerializedName("error")
    @Expose
    val errorData: ErrorData? = null
}
